vault.gemspec

```diff
@@ -19,6 +19,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "net-http-persistent", "~> 3.0"
```

Comment:
I have had really bad experiences managing gems that are client libraries which consume upstream libs like "restclient", etc. Adding this gem as a requirement means anyone who depends on this gem also must satisfy the same constraint. IMO, client libraries should only ever consume the stdlib for portability.

Especially given the context of vault-rails, it's highly probable to have conflicting constraints on a dependency like net-http-persistent in a large rails app.